packages/components/src/templates/next/components/internal/BaseParagraph/BaseParagraph.tsx

```diff
@@ -10,7 +10,7 @@ export const BaseParagraph = ({
 
   return (
     <p
-      className={`[&:not(:first-child)]:mt-6 sm:[&:not(:first-child)]:mt-7 [&:not(:last-child)]:mb-6 after:[&_a[target*="blank"]]:content-['_↗'] [&_a]:text-hyperlink [&_a]:underline visited:[&_a]:text-purple-600 hover:[&_a]:text-hyperlink-hover ${className}`}
+      className={`[&:not(:first-child)]:mt-6 [&:not(:last-child)]:mb-6 after:[&_a[target*="blank"]]:content-['_↗'] [&_a]:text-hyperlink [&_a]:underline visited:[&_a]:text-purple-600 hover:[&_a]:text-hyperlink-hover ${className}`}
```

Comment:
Do we not want to change the tokens for hyperlinks? `text-hyperlink` is no longer a token in the new color tokens. Same for `text-purple-600` and `text-hyperlink-hover`.

Reply:
@karrui we don't have the tokens for hyperlink colours yet right? i can add them in first